src/OrleansRuntime/Silo/Silo.cs

```diff
@@ -1054,6 +1038,16 @@ internal void SetMaxForwardCount_ForTesting(int val)
                 silo.OrleansConfig.Globals.MaxForwardCount = val;
             }
 
+            internal void LatchIsOverloaded(bool overloaded)
+            {
+                this.silo.Metrics.LatchIsOverload(overloaded);
```

Comment:
Yes, but I didn't touch it to avoid unnecessary changes. I can fix if you prefer